src/Microsoft.AspNet.Server.Kestrel/Filter/SocketInputStream.cs

```diff
@@ -5,6 +5,7 @@
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
+using IllyriadGames.ByteArrayExtensions;
```

Comment:
(sorry for continuing here)
Switching to .NET 4.6 will break current Mono users. The switch to the .NET 4.6 reference source only happened [back in October](https://github.com/mono/referencesource/commits/mono/mscorlib/system/array.cs), so all versions of Mono (including the most recent 4.2 and the mostly-working 4.0) don't have this member. So things will go back the way they were in beta2: aspnet/dnx#1105